spec/file_list_spec.rb

```diff
@@ -62,13 +65,13 @@
     end
 
     it 'should not include current working dir and parent' do
-      expect(subject.all).to eq(['file1'])
+      expect(subject.all.first.name).to eq(file_1.name)
     end
   end
 
   context 'when path directory contains file and directory' do
     before(:each) do
-      allow(Dir).to receive(:foreach).and_yield('file1').and_yield('a_directory')
+      allow(Dir).to receive(:foreach).and_yield(file_1.name).and_yield('a_directory')
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.